code/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+buildscript {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+        maven { url = uri("https://jitpack.io") }
+        mavenLocal()
+    }
+
+
+    dependencies {
+        classpath("com.github.adobe:aepsdk-commons:9deb721db7")
```

Comment:
nit: Can we add a note here for us to know and update the commit id for any changes to the plugin.